 
Data Hora: 19/03/2018 11:56:38
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:56:43
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:57:13
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:57:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:57:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:58:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:58:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:58:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:59:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:59:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 11:59:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:0:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:0:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:0:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:1:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:1:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:1:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:2:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:2:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:2:54
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:3:14
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:3:34
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:3:58
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:4:18
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:4:38
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:4:58
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:5:19
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:5:38
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:5:58
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:6:18
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:6:38
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:6:59
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:7:18
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:7:38
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 12:7:58
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/terminal
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.SalvarByTerminal(HttpRequestMessage request, DocumentosaidaView entidadePostada) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 340

--------------------------------------------
 
Data Hora: 19/03/2018 18:1:46
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 18:3:53
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:40:56
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:45:43
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:46:53
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:51:34
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:52:38
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:53:30
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:54:15
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:54:41
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 21:56:27
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:3:41
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:5:45
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:7:24
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:8:25
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:10:47
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:11:49
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:13:9
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:14:37
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:15:46
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:17:37
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:21:32
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:28:18
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 19/03/2018 22:29:31
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
